tests/distribution/pom.xml

```diff
@@ -285,7 +285,7 @@
                                 <delete dir="target/p2-repo"/>
                                 <!-- Figure out a way to delete tmp.* files -->
                                 <delete file="target/tmp"/>
-                                <delete dir="target/carbon-security-caas"/>
+                                <delete dir="target/carbon-security-caas" quiet='true'/>
```

Comment:
Please revert this change once you find the root cause for the ant running issue.